piperider_cli/__init__.py

```diff
@@ -216,6 +219,7 @@ def silent_none(var):
         return var
 
     env.globals['env_var'] = env_var
+    env.globals['Dimension'] = dimension
```

Comment:
Is it better not to share the template render logic from dbt config and metric filter?

Reply:
yes, agree with you that it's better to separate